packages/runtime/container-runtime/src/gc/gcConfigs.ts

```diff
@@ -107,32 +102,24 @@ export function generateGCConfigs(
 	);
 
 	/**
-	 * Whether sweep should run or not. This refers to whether Tombstones should fail on load and whether
-	 * sweep-ready nodes should be deleted.
+	 * Whether sweep should is enabled for this session or not. This refers to whether Tombstones should fail on load
```

Comment:
The sentence doesn't read well. Suggest rephrasing - " Whether sweep ~**should**~ is enabled for this session or not. **Sweep enablement would mean** Tombstones should fail on load and sweep-ready nodes should be deleted."

Plus - I'd recommend not narrating the 'how!' in the setting-declaration. The first half of the sentence is enough to describe the flag `sweepEnabled`. Rest is implementation detail of sweep, and should be left to be described near the business logic of sweep.

Reply:
I rephrased it as per your suggestion.

For the how part, this is internal GC configs and is where we implement how these configs are generated. So, I believe this is the right place to have these comments. After this point, GC simply uses this configs.